src/generated_struct_info32.json

```diff
@@ -185,6 +185,7 @@
         "EMSCRIPTEN_FULLSCREEN_SCALE_CENTER": 3,
         "EMSCRIPTEN_FULLSCREEN_SCALE_DEFAULT": 0,
         "EMSCRIPTEN_FULLSCREEN_SCALE_STRETCH": 1,
+        "EMSCRIPTEN_ORIENTATION_UNSUPPORTED": 0,
```

Comment:
I believe the 64-bit version of this file needs updating too (
Run `tools/maint/gen_struct_info.py --wasm64`)